### PR TITLE
fix: Update links to in-toto attestation spec

### PR DESCRIPTION
**Pull request by ianlewis**
Signed-off-by: Ian Lewis <ianlewis@google.com>

### CODE REVIEW

File: docs/provenance/v1-rc1.md [hunk 1]
```diff
@@ -148,7 +148,7 @@ REQUIRED for SLSA Build L1: `buildType`, `externalParameters`
 <tr><th>Field<th>Type<th>Description
 
 <tr id="buildType"><td><code>buildType</code>
-<td>string (<a href="https://github.com/in-toto/attestation/blob/main/spec/field_types.md#TypeURI">TypeURI</a>)<td>
+<td>string (<a href="https://github.com/in-toto/attestation/blob/main/spec/v1.0-draft/scalar_field_types.md#typeuri">TypeURI</a>)<td>
```
Comment (ianlewis) [inline docs/provenance/v1-rc1.md:151]: Not sure if SLSA v1 should be pointing to the v1.0-draft for in-toto attestations but I assumed so.

**Review by MarkLodato — APPROVED** [review]
LGTM but it would be good to have another reviewer as well

**Comment by TomHennen** [comment]
I think all the various _type fields in 1.0 should be updated to point to `"_type": "https://in-toto.io/Statement/v1"`

It looks like there are a number of these in various files?

**Comment by ianlewis** [comment]
> I think all the various _type fields in 1.0 should be updated to point to `"_type": "https://in-toto.io/Statement/v1"`
> 
> It looks like there are a number of these in various files?

I updated this in the doc files related to SLSA v1 (and v1-rc1) (including cue files).

File: docs/provenance/v1-rc1.md (hunk 1 above)
Comment (marcelamelara) [reply docs/provenance/v1-rc1.md:151]: Thanks for these updates! We're about to do a tagged release of v1 in-toto attestations (by EOW), so the SLSA docs can probably be updated to point to those instead.

**Comment by joshuagl** [comment]
Thanks for working on this! there's a mixture of v0.1.0 and v1.0-draft links, per @marcelamelara's comment https://github.com/slsa-framework/slsa/pull/679/files#r1135698487 can we update this PR to only link to v1 once the release is made (some time this week)?

**Comment by ianlewis** [comment]
> Thanks for working on this! there's a mixture of v0.1.0 and v1.0-draft links, per @marcelamelara's comment https://github.com/slsa-framework/slsa/pull/679/files#r1135698487 can we update this PR to only link to v1 once the release is made (some time this week)?

The mixture was intentional as I believe that SLSA v0.2 (i.e. prior to 1.0) would be technically be using the v0.1.0 versions of the in-toto attestation spec.

I can wait on the PR until the v1 is finalized and update the v1 links then.

**Comment by joshuagl** [comment]
> > Thanks for working on this! there's a mixture of v0.1.0 and v1.0-draft links, per @marcelamelara's comment https://github.com/slsa-framework/slsa/pull/679/files#r1135698487 can we update this PR to only link to v1 once the release is made (some time this week)?
> 
> The mixture was intentional as I believe that SLSA v0.2 (i.e. prior to 1.0) would be technically be using the v0.1.0 versions of the in-toto attestation spec.

Excellent point, that sounds right to me.

> I can wait on the PR until the v1 is finalized and update the v1 links then.

Thank you!

**Comment by ianlewis** [comment]
v1 for the in-toto spec was finalized so I hope to clean this up by EOW

**Comment by ianlewis** [comment]
@MarkLodato @TomHennen Updated and merged with HEAD. PTAL

File: docs/provenance/schema/v1/provenance.cue [hunk 2]
```diff
@@ -4,7 +4,7 @@
     "subject": [...],
 
     // Predicate:
-    "predicateType": "https://slsa.dev/provenance/v1?draft",
+    "predicateType": "https://slsa.dev/provenance/v1",
```
Comment (MarkLodato) [inline docs/provenance/schema/v1/provenance.cue:7]: Let's hold off for now - we'll do this when we mark v1.0 as final

File: docs/attestation-model.md [hunk 3]
```diff
@@ -96,7 +96,7 @@ recognize that other choices MAY be necessary in various cases.
 | Bundle | **[JSON Lines]**, see [attestation bundle] |
 | Storage/Lookup | **TBD** |
 
-[attestation bundle]: https://github.com/in-toto/attestation/blob/main/spec/bundle.md
+[attestation bundle]: https://github.com/in-toto/attestation/blob/main/spec/v0.1.0/bundle.md
```
Comment (MarkLodato) [inline docs/attestation-model.md:99]: why not v1?

**Review by MarkLodato — CHANGES_REQUESTED** [review]
Thanks, Ian! Just two small fixes

File: docs/attestation-model.md (hunk 3 above)
Comment (MarkLodato) [reply docs/attestation-model.md]: pushed a change to fix

File: docs/provenance/schema/v1/provenance.cue (hunk 2 above)
Comment (MarkLodato) [reply docs/provenance/schema/v1/provenance.cue]: pushed a fix

**Review by MarkLodato — APPROVED** [review]

**Review by marcelamelara — APPROVED** [review]
Thanks!

**Comment by ianlewis** [comment]
Thanks for fixing those.

**Pull request merged**